packages/configuration-development/README.md

```diff
@@ -46,8 +51,7 @@ and it will be as if you had this:
 }
 ```
 
-If you prefer to use another formatter (or any other hint/connector,
-etc.) you can do something like:
+The following code snippet includes another formatter \(or any other hint/connector, and so on\).
```

Comment:
Reserved characters that are located in the body text should be escaped to avoid issues with formatting.  

```md
\   backslash
`   backtick
*   asterisk
_   underscore
{}  curly braces
[]  square brackets
()  parentheses
#   hash mark
+   plus sign
-   minus sign (hyphen)
.   dot
!   exclamation mark
```